src/base/types.h

```diff
@@ -35,7 +35,7 @@ const qlonglong MAX_ETA = 8640000;
 
 enum class ShutdownAction
 {
-    None,
+    Exit,
```

Comment:
>The old identifiers were saying that the system will be suspended in addition to the exit.

I assume you mean the *new* identifier?

-
maybe your concern is because the bad naming of the enum class name, could something like `ShutdownDialogAction` better?

btw, I don't read `Suspend` is going to do anything more than just suspending the system. For me, those actions are orthogonal (well, except Shutdown & Exit).

Unless you want to combine actions, there might be a point expanding this enum...